dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/datasource/mysql/MysqlDatasourceProcessor.java

```diff
@@ -51,7 +51,7 @@
 
     private static final String ALLOW_URL_IN_LOCAL_IN_FILE_NAME = "allowUrlInLocalInfile";
 
-    private static final String APPEND_PARAMS = "allowLoadLocalInfile=false&autoDeserialize=false&allowLocalInfile=false&allowUrlInLocalInfile=false";
+    private static final String APPEND_PARAMS = "allowLoadLocalInfile=false&autoDeserialize=false&allowLocalInfile=false&allowUrlInLocalInfile=false&useSSL=false";
```

Comment:
I am not sure set `useSSL=false` here is reasonable.

Reply:
Okay, let me try without adding it for now

Reply:
Please rebase the upstream code and resolve the conflicts.

Reply:
already solved

Reply:
@lyggyhmm Everything looks good, one minor may need to change.
Do we require adding  `useSSL=false` in `datasource.properties` ? I worried that some new user may meet problem here.

Reply:
The MySQL 8.X version does not seem to throw any warnings on the useSSL configuration, which has been tested on my own server. Of course, my knowledge is limited, I want to know what insights you have

Reply:
@ruanwenjun hi,the MySQL 8.X version does not seem to throw any warnings on the useSSL configuration, which has been tested on my own server. Of course, my knowledge is limited, I want to know what insights you have

Reply:
Of course, configuring useSSL=false should be more secure, after all, we do not use SSL certification

Reply:
I have removed the useSSL=false configuration last week

Reply:
That's ok.